# SequiturData.msg

Header header
geometry_msgs/Pose pose
geometry_msgs/Accel accel
geometry_msgs/Vector3 magnetic_field
